lib/shellable.ts

```diff
@@ -145,23 +146,6 @@ export interface AssumeRole {
   externalId?: string;
 }
 
-/**
- * Properties used to create a Shellable
- */
-export interface ShellableProps extends ShellableOptions {
```

Comment:
What’s up here? No props?

Reply:
Duplicate definition.

Reply:
Not sure what that means? What am I missing?

Reply:
We had 2 definitions of this interface in the same file. They were both exactly the same.

Reply:
👍